app/pathogen/arbovirus/dashboard/(map)/arbo-map-cluster-utils.tsx

```diff
@@ -0,0 +1,224 @@
+import { pathogenColors } from "@/app/pathogen/arbovirus/dashboard/(map)/ArbovirusMap";
```

Comment:
This file contains a bunch of utility functions that aid in building the cluster markers. The solution here was directly adopted from here - https://docs.mapbox.com/mapbox-gl-js/example/cluster-html/ feel free to give it a read. Had to adopt the solution to use react-map-gl which works a little differently.